src/Inputs.ts

```diff
@@ -136,6 +137,14 @@ export class CoreInputs implements Inputs {
         return generate == 'true'
     }
 
+    get makeLatest(): boolean {
```

Comment:
I think this should be a string input so we can select true, false or legacy.

Reply:
I updates this to return `undefined` if the option wasn't provided. Not sure if that's what you want or if it instead should return `"legacy"` :thinking:

Reply:
Hmm, not sure - my guess would be undefined == legacy, so either would probably be sufficient. I can test with my playground repo and we can always change the default if it's wrong after.